notebooks/transport/http.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 const (
-	prefixNotebooks = "/api/v2private/notebooks"
+	prefixNotebooks     = "/api/v2private/notebooks"
+	allNotebooksJSONKey = "flows"
```

Comment:
Small update to the HTTP API - the server response actually needs to be a JSON object with a key "flows" and the value being an array of all of the notebooks. Previously I had it just returning an array with the objects, which the frontend is not setup for.